x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/document_creation_buttons.tsx

```diff
@@ -45,90 +52,147 @@ export const DocumentCreationButtons: React.FC<Props> = ({ disabled = false }) =
   useEffect(() => {
     switch (method) {
       case 'json':
-        openDocumentCreation('file');
+        openDocumentCreation('json');
         break;
       case 'api':
         openDocumentCreation('api');
         break;
+      case 'elasticsearchIndex':
+        openDocumentCreation('elasticsearchIndex');
+        break;
     }
   }, []);
 
   const crawlerLink = generateEnginePath(ENGINE_CRAWLER_PATH);
 
-  return (
+  const helperText = (
+    <p>
+      {i18n.translate('xpack.enterpriseSearch.appSearch.documentCreation.helperText', {
+        defaultMessage:
+          'There are four ways to send documents to your engine for indexing. You can paste or upload a JSON file, POST to the documents API endpoint, connect to an existing Elasticsearch index, or use the  Elastic Web Crawler to automatically index documents from a URL.',
+      })}
+    </p>
+  );
+
+  const emptyState = (
+    <EuiFlexItem>
+      <EuiEmptyPrompt
+        icon={<EuiImage size="fullWidth" src={illustration} alt="" />}
```

Comment:
Does this behave properly for accessibility ? Would screenreaders get the title and say nothing on icon?

Reply:
Nice catch. It's hard to label this well since it's a bit abstract, but I can certainly add "Illustration" as alt text

Reply:
https://github.com/elastic/kibana/pull/127162/commits/f8efd09e8178a44eec8167b70f7ecb8486bf24f8